client/blocks/inline-help/contextual-help.js

```diff
@@ -966,6 +966,50 @@ const contextLinksForSection = {
 			),
 		},
 	],
+	domains: [
+		{
+			link: localizeUrl( 'https://en.support.wordpress.com/add-email/' ),
+			post_id: 34087,
+			title: translate( 'Add Email to your Domain' ),
+			description: translate(
+				'Want to use a custom email with your domain, such as info@yourgroovydomain.com? There are multiple ways to add email to your custom domain.'
+			),
+		},
+		{
+			link: localizeUrl( 'https://en.support.wordpress.com/domains/custom-dns/' ),
+			post_id: 6595,
+			title: translate( 'Manage Custom DNS' ),
+			description: translate(
+				'Custom DNS records are special settings that change how your domain works. They allow you to connect your domain to third-party services that are not hosted on WordPress.com, such as an email provider.'
+			),
+		},
+		{
+			link: localizeUrl(
+				'https://en.support.wordpress.com/move-domain/transfer-domain-registration/'
+			),
+			post_id: 41298,
+			title: translate( 'Transfer a Domain to Another Registrar' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Transfer Domain To Another Registrar' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/contextual-help-for-domains)